frontend/src/Editor/LeftSidebar/SidebarPageSelector/index.jsx

```diff
@@ -46,6 +47,11 @@ const LeftSidebarPageSelector = ({
     shallow
   );
 
+  const sortedAllPages = useMemo(
```

Comment:
You can move this function outside of this component. No need to use useMemo and also `allpages` is an array.